build_modules/CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 0.4.0
+
+### Improvements
+
+- Modules are now platform specific, and config specific imports are supported,
+  but only for `dart.library.*` constants.
```

Comment:
I think this is already an constraint on conditional imports in the CFE (though it's maybe not documented super well?)

I don't think we should phrase this as if it's a limitation. "config specific imports against `dart.library.*` constants are supported"

Reply:
done

Reply:
> though it's maybe not documented super well?

or at all ;)